lib/jlcpcb.test.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, test, describe } from "bun:test"
```

Comment:
rename this test file to correspond with what it's actually testing, which is the new unique quantity format